orttraining/orttraining/eager/test/ort_ops.py

```diff
@@ -232,6 +232,17 @@ def test_softmax(self):
         ort_result = torch.softmax(ort_tensor, dim=1)
         assert torch.allclose(cpu_result, ort_result.cpu())
 
+    def test_log_softmax(self):
+        device = self.get_device()
+        cpu_tensor = torch.rand(3, 5)
+        ort_tensor = cpu_tensor.to(device)
+        cpu_result_a = torch.log_softmax(cpu_tensor, dim=1)
+        ort_result_a = torch.log_softmax(ort_tensor, dim=1)
+        assert torch.allclose(cpu_result_a, ort_result_a.cpu())
+        cpu_result_b = torch.log_softmax(cpu_tensor, dim=0)
+        ort_result_b = torch.log_softmax(ort_tensor, dim=0)
```

Comment:
Consider adding case with dim = -1 to exercise the dim < 0 code path or increase the test tensor dim to 3 (rand(3,5,5 for example) update the dim=0 case to dim =-1.

Reply:
also test the out param